Sources/Datadog/DatadogCore/Storage/Writing/FileWriter.swift

```diff
@@ -21,12 +21,25 @@ internal struct FileWriter: Writer {
     // MARK: - Writing data
 
     /// Encodes given value to JSON data and writes it to the file.
```

Comment:
We could update the comment header to reflect the change

Reply:
yes, fixed it.